python/cudf_polars/cudf_polars/dsl/expressions/datetime.py

```diff
@@ -104,6 +104,12 @@ def from_polars(cls, obj: pl_expr.TemporalFunction) -> Self:
         Name.Nanosecond: plc.datetime.DatetimeComponent.NANOSECOND,
     }
 
+    _valid_ops: ClassVar[list[Name]] = [
```

Comment:
Nit: Could we make this a `set`?

Reply:
Done. Lets merge this one and I'll propagate the changes to the other open datetime PRs.